### ISSUE TITLE
Reply not working

### ISSUE THREAD

**Issue by p-g-krish** [Viima/jquery-comments#94]
When i click the reply button, It was not add the text box . In the firefox console, its shows nothing . But in the chrome console . Its shows the 
**jquery-comments.js:2096 The given range isn't in document.  
      moveCursorToEnd @ jquery-comments.js:2096**

Especially on this moveCursorToEnd section

sel.addRange(range);

**Comment by luiscovar** [comment]
Same issue here. Parents is being saved as an empty string instead of 'null' without quotes and is causing errors. Not sure how to fix this.

**Comment by lucassimao** [comment]
Someone was solved this? it works on the demo, but for me is not working. I already enabled the reply through enableReplying options but doesn't works

**Comment by burzum** [comment]
I don't have this problem using the latest release. Works for me.

**Comment by yenwushu** [comment]
In my case, if the comment is not a child, "parent" must be "null". I got error by setting "parent: 0". That's the reason for "the given range isn't in document".

**Comment by lucassimao** [comment]
Actually, the error was in my code! 
At the backend  i'm using Mongodb and in the Mongodb world, the primary key for some document (in this case, any new comment) it's a field named '_id' of type ObjectId .  This field stores a value that is a mix of numers and letters, like a hash. Although i've mapped this '_id' field to the id field expected by the jquery-comments library using the fieldMappings configuration option, the jquery-comments library assumes that the value is a int number .... when the library tried to use this 'id' value as an integer to associate some child comment with his parent everything screwed up 

The bottom line: make sure you are storing the 'id' and 'parent' comment fields as valid integers (as the documentation says) and any doubt you can see how the library expects the data in the demo's github data/comments-data.js

**Comment by pasupathy** [comment]
do not include the parent field in your insert, else it will insert as zero
